engine/platform/posix/sys_posix.c

```diff
@@ -145,6 +146,16 @@ void Posix_Daemonize( void )
 
 }
 
+static void Posix_SigtermCallback( int signum )
+{
+	Sys_Quit();
+}
+
+void Posix_SetupSigtermHandling( void )
+{
+	signal( SIGTERM, Posix_SigtermCallback );
```

Comment:
Better use `sigaction(2)`, it's more portable than `signal(2)`.

Reply:
The example is in `crashhandler.c` btw.

Reply:
Ok, I did it.